modules/core/src/main/java/org/apache/ignite/internal/processors/metric/list/AbstractMonitoringList.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.metric.list;
+
+import org.apache.ignite.spi.metric.list.MonitoringList;
+import org.apache.ignite.spi.metric.list.MonitoringRow;
+import org.apache.ignite.spi.metric.list.MonitoringRowAttributeWalker;
+
+/**
+ *
+ */
+abstract class AbstractMonitoringList<R extends MonitoringRow> implements MonitoringList<R> {
+    /** Name of the list. */
+    private final String name;
+
+    /** Description of the list. */
+    private final String description;
+
+    /** Class of the row */
+    private final Class<R> rowClass;
+
+    /**
+     * Row attribute walker.
+     *
+     * @see org.apache.ignite.codegen.MonitoringRowAttributeWalkerGenerator
```

Comment:
IDEA highlighted this line as an error, since MonitoringRowAttributeWalkerGenerator from module `ignite-codegen` which is not in dependencies.

Reply:
So it's the issue in the IDEA, isn't it?)

Reply:
Not sure it's IDEA issue, perhaps we shouldn't link to unreachable classes even in JavaDoc.

Reply:
Or at least you can use `@see "text"` syntax, but not `@see reference`